docs/server/source/cloud-deployment-templates/node-on-kubernetes.rst

```diff
@@ -155,9 +155,53 @@ Initially, the status of persistent volume claims might be "Pending"
 but it should become "Bound" fairly quickly.
 
 
+Step 5: Create the Config Map - Optional
+----------------------------------------
+
+This step is required only if you are planning to set up multiple
+`BigchainDB nodes
+<https://docs.bigchaindb.com/en/latest/terminology.html#node>`_, else you can
+skip to the :ref:`next step <Step 6: Run MongoDB as a StatefulSet>`.
+
+MongoDB reads the local ``/etc/hosts`` file while bootstrapping a replica set
+to resolve the hostname provided to the ``rs.initialize()``
+command. It needs to ensure that the replica set is being initialized in the
+same instance where the MongoDB instance is running.
+
+To achieve this, we create a ConfigMap with the FQDN of the MongoDB instance
+and populate the ``/etc/hosts`` file with this value so that a replica set can
+be created seamlessly.
+
+Get the file ``mongo-cm.yaml`` from GitHub using:
+
+.. code:: bash
+
+   $ wget https://raw.githubusercontent.com/bigchaindb/bigchaindb/master/k8s/mongodb/mongo-cm.yaml
+
+You may want to update the ``data.fqdn`` field in the file before creating the
+ConfigMap.
+This name should resolve to the MongoDB instance in your cluster.
```

Comment:
I have covered the scenario for Azure and other providers in the latest commit.
If they are running a MongoDB instance already, they are following the wrong documentation. One needs to follow the `add-node-on-kubernetes` doc, in which scenario the new MongoDB instance isn't still up and running, and one has to pre-allocate a DNS name for the instance coming up.